README.md

```diff
@@ -42,25 +51,20 @@ It will output the `.vsix` file to:
 From there, you can share and/or install the `.vsix` file as desired. Here is a decent [blog post about working with `.vsix` files manually](https://weblog.west-wind.com/posts/2016/Mar/01/Registering-and-Unregistering-a-VSIX-Extension-from-the-Command-Line#Installing)
 
 #### Using the Extension with Changes
-
 Once you have the Visual Studio environment with the modified extension loaded, or have installed the extension from the `.vsix` file, you can test it by opening the example project:
 
 	Conan.VisualStudio.Examples/ExampleCLI/ExampleCLI.sln
 
-This example has a `conanfile.txt` with a dependency on the popular formatting library `fmt`.  When the project is first opened, Visual Studio is unable to find the `fmt` project dependency.  The extension should immediately run however, and Visual Studio should give you a toolbar dropdown message offering to refresh the project.  After refreshing, the `fmt` headers should be found by the compiler, and the `lib` files should be found by the linker successfully.  
-
-
-Update 03-14-2019
-------------------
-Thanks to @sboulema and @SSE4, the extension now provides the minimum required functionality for basic use and initial testing. 
-
-Update 02-06-2019
-------------------
-Plugin development is now being resumed after being stagnant for the past year. 
-
-The overall goal of the extension is for Visual Studio to be able to execute Conan automatically as-needed based on the currently loaded solution/project/configuration.  Over time, this could grow to a lot of convenience operations.  However, the primary (first) objective is to run `conan install` which will generate `conanbuildinfo.props` and satisfy the dependencies. 
+This example has a `conanfile.txt` with a dependency on the popular formatting library `fmt`. When the project is first opened, Visual Studio is unable to find the `fmt` project dependency. The extension should immediately run however, and Visual Studio should give you a toolbar dropdown message offering to refresh the project. After refreshing, the `fmt` headers should be found by the compiler, and the `lib` files should be found by the linker successfully.  
```

Comment:
this is no longer the true, there is no toolbsar dropdown and there is no need to reload the project